src/coreclr/gc/gc.cpp

```diff
@@ -8815,87 +8871,96 @@ bool gc_heap::on_used_changed (uint8_t* new_used)
     return true;
 }
 
-bool gc_heap::inplace_commit_card_table (uint8_t* from, uint8_t* to)
+bool gc_heap::get_card_table_commit_layout (uint8_t* from, uint8_t* to, uint8_t* commit_begins[total_bookkeeping_elements], size_t commit_sizes[total_bookkeeping_elements], size_t new_sizes[total_bookkeeping_elements])
```

Comment:
nit - could we rename new_sizes to new_commit_sizes? and also make this line not so long?